webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
+      <ul :class="connectionsClass">
+        <li
+          v-for="connection in filteredConnections"
+          :key="connection.id"
+          class="connections__item"
+        >
+          <user-teaser :user="connection" />
+        </li>
+      </ul>
+      <base-button
+        v-if="hasMore"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-31T12:01:31Z" title="Tuesday, March 31st 2020, 2:01:31 pm +02:00">Mar 31, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,133 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <ds-text tag="h5" color="soft" class="spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
+      <div :class="connectionsClass">
+        <user-teaser
+          v-for="connection in filteredConnections"
+          class="spacer-x-small"
+          :user="connection"
+          :key="connection.id"
+        />
+      </div>
+      <base-button
+        v-if="allConnectionsCount - connections.length"
+        @click="$emit('fetchAllConnections', type)"
```


we have been in the habit of putting the `events` at the bottom of the list.

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T05:09:53Z" title="Wednesday, April 1st 2020, 7:09:53 am +02:00">Apr 1, 2020</time>_

---


I bumped into that question, :smile: 
Just apply [Vue's styleguide regarding attribute order](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended)? (which does put events at the 2nd last place, yes)

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:01:10Z" title="Wednesday, April 1st 2020, 8:01:10 am +02:00">Apr 1, 2020</time>_

---


yes, please!

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T07:04:01Z" title="Wednesday, April 1st 2020, 9:04:01 am +02:00">Apr 1, 2020</time>_

---


done in 8e98570